src/app/pages/features.html

```diff
@@ -101,16 +104,16 @@ <h1 class="uppercase">See OpenShift.io in Action</h1>
           <img src="../../assets/images/screencap_home.png" alt="Dashboard Screenshot" width="100%" />
         </div>
         <div>
-          <h2>WHAT IS OPENSHIFT.IO (OSIO)?</h2>
+          <h2>WHAT IS OPENSHIFT.IO?</h2>
           <p>
-            OpenShift.io, combined with OpenShift Online, provides an integrated approach to DevOps, including all the tools a team needs to analyze, plan, create and deploy services.
+            OpenShift.io provides an integrated approach to DevOps with all the tools a team needs to analyze, plan, create, and deploy services. Main features include:
           </p>
           <p>
             <ul>
               <li>One-click Linux container management</li>
-              <li>Machine learning system</li>
-              <li>Open Source</li>
-              <li>Incorporate many projects like fabric8 and Eclipse Che</li>
+              <li>Integrations with open source projects like fabric8, Eclipse Che, and OpenShift Online</li>
```

Comment:
@stacymcauliffe I'm not sure "Integrations" is the right word for this.  "Utilizes many open source projects like..." possibly?

Reply:
Works! @mindreeper2420 - "Utilizes many open source projects like fabric8, Eclipse Che, and OpenShift Online."